jupyterhub_moss/templates/option_form.html

```diff
@@ -197,6 +207,40 @@ <h4 style="text-align: center">List of available resources:</h4>
       name="jupyterlab"
       value="true"
     />
+    <label>Jupyter environment:</label>
+    <div id="jupyter_environments" class="form-offset-row">
+      <div id="jupyter_environments_default"></div>
+      <div>Custom:</div>
+      <div id="jupyter_environments_custom"></div>
+      <div class="environment-div">
+        <input type="hidden" value="custom"/>
+        <input
+          type="radio"
+          id="environment_add_radio"
+          name="environment_path"
+          value=""
+          disabled
+        />
+        <input
+          type="text"
+          id="environment_add_name"
+          placeholder="Environment name"
+        />
+        <input
+          type="text"
+          id="environment_add_path"
+          placeholder="/path/to/jupyter/env/bin"
```

Comment:
I would just add ` *` at the end of this placeholder to convey the mandatory nature of this field.